### PR TITLE
ddt removal in cli env tests

### PR DESCRIPTION
**Pull request by sthirugn**
```sh
# nosetests tests/foreman/cli/test_environment.py:TestEnvironment.test_update_positive
.
----------------------------------------------------------------------
Ran 1 test in 12.654s

OK

# nosetests tests/foreman/cli/test_environment.py:TestEnvironment.test_update_negative
.
----------------------------------------------------------------------
Ran 1 test in 19.724s

OK
```

### CODE REVIEW

File: tests/foreman/cli/test_environment.py [hunk 1]
```diff
@@ -11,6 +10,15 @@
 from robottelo.test import MetaCLITestCase
 
 
+def valid_names():
+    """Returns a list of valid environment names"""
+    return [
+        gen_string('alpha', 10),
+        gen_string('numeric', 10),
+        gen_string('alphanumeric', 10),
```
Comment (JacobCallahan) [inline tests/foreman/cli/test_environment.py]: i believe 10 is the default length. not a blocker

**Comment by JacobCallahan** [comment]
ACK

File: tests/foreman/cli/test_environment.py (hunk 1 above)
Comment (sthirugn) [inline tests/foreman/cli/test_environment.py]: :+1:

**Comment by sthirugn** [comment]
Updated and rebased

**Comment by elyezer** [comment]
ACK

**Comment by sthirugn** [comment]
Part of https://github.com/SatelliteQE/robottelo/issues/2769

**Pull request merged**